easybuild/easyconfigs/y/YALES2-deps/YALES2-deps-2022-foss-2021a.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'Bundle'
+
+name = 'YALES2-deps'
+version = '2022'
+
+homepage = 'https://www.coria-cfd.fr/index.php/YALES2'
+description = """YALES2 aims at the solving of two-phase combustion from primary atomization to
+pollutant prediction on massive complex meshes. It is able to handle
+efficiently unstructured meshes with several billions of elements, thus
+enabling the Direct Numerical Simulation of laboratory and semi-industrial
+configurations."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CMake', '3.20.1'),
+    ('CWIPI', '0.12.0'),
+    ('git', '2.32.0', '-nodocs'),
+    ('HDF5', '1.10.7'),
+    ('Hypre', '2.21.0'),
+    ('METIS', '5.1.0'),
+    ('Mmg', '5.6.0'),
+    ('PETSc', '3.15.1'),
+    ('SLEPc', '3.15.1'),
+    ('SCOTCH', '6.1.0'),
+    # needed for python bindings
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('f90wrap', '0.2.8'),
+    ('h5py', '3.2.1'),
+    ('matplotlib', '3.4.2'),
+]
+
+components = [
+    # add configuration files for the easybuild platform
+    ('yales2-easybuild-platform', version, {
+        'easyblock': 'CmdCp',
+        'sources': [{'filename': 'YALES2-deps-%(version)s_easybuild-platform.patch', 'extract_cmd': "cp %s ."}],
+        'checksums': ['66927b67e7fff01f023e2ba30ad4a58644503174fe1b1ae3ec8ca6a9063982fa'],
+        'cmds_map': [('.*', "git apply %(source)s")],
+        'files_to_copy': ['platforms'],
+    }),
+]
+
+_platform_files = ['easybuild.bash.env', 'easybuild.include.f90', 'easybuild.Makefile.in',
+                   'easybuild.slurm.run', 'easybuild.tools_params.sh']
+
+sanity_check_paths = {
+    'files': ['platforms/%s' % x for x in _platform_files],
+    'dirs': [],
+}
+
+# User's working directory with YALES2 source code
+# _yales2_home = '$HOME'
+_yales2_home = 'VSC_SCRATCH'
+
+modextravars = {
+    'YALES2_HOSTTYPE': 'easybuild',
+    'YALES2_ARCH': '-march=native',
+    'Y2_COMPILE_MODE': '2_STAGES',
+    'Y2_TARGET': 'DYNAMIC',
+    'COMPILER': 'gcc',
+}
+
+modluafooter = """
```

Comment:
Do we also want a `modtclfooter` equivalent?

Can we avoid having to use `modluafooter` if we implement support in framework to define environment variable using already defined environment variables?